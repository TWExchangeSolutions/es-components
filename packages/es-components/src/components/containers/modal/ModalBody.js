import styled from 'styled-components';

// Note: ModalBody relies on a parent (Modal) with ThemeProvider wrapping it
const ModalBody = styled.div`
  color: ${props => props.theme.colors.gray9};
  padding: 15px;
`;

export default ModalBody;
