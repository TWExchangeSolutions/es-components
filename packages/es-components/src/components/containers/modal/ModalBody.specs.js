/* eslint-env jest */

import React from 'react';
import renderer from 'react-test-renderer';
import { ThemeProvider } from 'styled-components';
import viaTheme from 'es-components-via-theme';

import ModalBody from './ModalBody';

describe('modal Body component', () => {
  let instanceToRender;

  beforeEach(() => {
    instanceToRender = (
      <ThemeProvider theme={viaTheme}>
        <ModalBody className="Body">This is the Body.</ModalBody>
      </ThemeProvider>
    );
  });

  it('renders as expected', () => {
    const tree = renderer.create(instanceToRender).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
