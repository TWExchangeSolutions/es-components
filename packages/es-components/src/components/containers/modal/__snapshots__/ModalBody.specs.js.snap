// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modal Body component renders as expected 1`] = `
<div
  className="Body sc-bdVaJa dVAtex"
>
  This is the Body.
</div>
`;
